apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/AssessmentQuestionManualGrading.html.tsx

```diff
@@ -75,6 +73,7 @@ function AssessmentQuestionManualGradingInner({
   questionTitle,
   questionNumber,
 }: AssessmentQuestionManualGradingInnerProps) {
+  const queryClient = useQueryClient();
```

Comment:
Well, this caught at least one bugged usage.

Reply:
Nice!